client/src/components/Form/Form.scss

```diff
@@ -53,9 +53,11 @@
   margin: $spacer 0 #{-$spacer};
 }
 
-// Remove underline and column offset
-// Used on places like tree batch actions
-.form--no-dividers .form-group {
+// Remove underline and column offset. Used on places like tree batch actions.
+// Apply form--no-dividers to a form element, or form-field--no-divider to an individal
+// form field.
+.form--no-dividers .form-group,
+.form-group.form-field--no-divider {
```

Comment:
nice one, I think `.form-group--no-divider` is sufficient, no need to nest/combo classes :D

Reply:
Fair enough, updated